tests/framework/cli/test_cli.py

```diff
@@ -200,7 +200,7 @@ def test_help(self):
         """Check that help output includes stub_cli group description."""
         cmd_collection = CommandCollection(("Commands", [cli, stub_cli]))
         result = CliRunner().invoke(cmd_collection, [])
-        assert result.exit_code == 0
+        assert result.exit_code in (0, 2)
```

Comment:
Should help command result in exit code 2?

Reply:
Oh sorry I missed that. I still think it's weird to return 2 when it's a normal to do `kedro --help`. I can check if there's a way around this that `click` suggests

Reply:
This was the change: https://github.com/pallets/click/pull/1489

From their release notes:

> If help is shown because no_args_is_help is enabled (defaults to True for groups, False for commands), the exit code is 2 instead of 0. [#1489](https://github.com/pallets/click/issues/1489)